core/server/common/src/main/java/alluxio/Process.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio;
+
+/**
+ * Interface representing an Alluxio process.
+ */
+public interface Process {
+
+  /**
+   * Starts the Alluxio process. This call may block until the process is stopped via
+   * {@link #stop()}. The {@link #waitForReady()} method can be used to make sure that the
+   * process is ready to serve requests.
+   *
+   * @throws Exception if the process fails to start
+   */
+  void start() throws Exception;
```

Comment:
throwing `Exception` is more convenient for implementors and less convenient for users of the method. We have more implementors than users, so I think throwing `Exception` is reasonable.

I'm going to change checkstyle so that it isn't required to javadoc exceptions. Then we don't have to worry about having redundant/inaccurate descriptions for general exceptions like `Exception`, `IOException`, or `AlluxioException`